pt.html

```diff
@@ -201,96 +207,97 @@
 <panel w=500 h=450 fadeInOn="flip_spaced_rep">
 <pic src="pics/intro0.png" sx=950 sy=0></pic>
 <words x=10 y=10 w=430 h=60>
-	And it's <i>awesome</i>.
-	I started using Spaced Repetition earlier this year to learn French.
+	E é <i>incrível</i>.
+	Comecei a usar o Repetição Espaçada no início deste ano para aprender francês.
+
 </words>
 <words x=30 y=350 w=430 h=60>
-	In two <i>months</i>, I learnt more words than I did in two <i>years</i>
-	of high school French classes.
+	Em dois <i>meses</i> eu aprendi mais palavras do que em dois <i>anos</i>
+	de aulas de francês no ensino médio.
```

Comment:
Não encaixa no balão.

Acho que dá para tirar `de aulas` que continua o mesmo sentido
```suggestion
	de francês no ensino médio.
```